contexts/did-v1.jsonld

```diff
@@ -16,7 +15,6 @@
     "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
     "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
     "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
-    "ServiceEndpointProxyService": "didv:ServiceEndpointProxyService",
```

Comment:
This is something that's important to Veres One, and I expect, other DID Methods once people fully grasp GDPR and how it impacts DID Method implementers. We're waiting for the Service Endpoint discussion to settle before proposing it. It's fine that it's removed now and awaits a definition of it, but I want to be crystal clear that removing it at this point DOES NOT imply that it was never here or that we do not intend to define it in DID Core.

/cc @dlongley @dmitrizagidulin

Reply:
this seems like something to discuss on a special topic call, about services.